services/userService.js

```diff
@@ -0,0 +1,51 @@
+const jwt = require('jsonwebtoken')
+
+const { userDao } = require('../models');
+
+
+const signIn = async (authCode) => {
+
+    const clientId = process.env.CLIENT_ID;
+    const redirect_uri = process.env.REDIRECT_URI;
+
+    const getToken = await axios({
+        url:`https://kauth.kakao.com/oauth/token`,
+        method: 'POST',
+        headers:{ 
+        'Content-type': 'application/x-www-form-urlencoded;charset=utf-8'
+        },
+        data : `grant_type=authorization_code&client_id=${clientId}&redirect_uri=${redirect_uri}&code=${authCode}`
+    });
+
+    const UserData = await axios({
```

Comment:
UserData는 클래스가 아니기에 컨벤션을 지켜 수정해주세요.
```suggestion
    const userData = await axios({
```

Reply:
수정했습니다.